File: apps/bare-expo/localModulesExamples/simpleModule.kt
package src.nested

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

public class simpleModule : Module() {
    override fun definition() = ModuleDefinition {
        Constant("test") { ->
            "Kotlin constant 1 new localModulesProvider"
        }
    }
}

File: apps/native-component-list/src/screens/LocalModules/localModulesExamples/SimpleModule.swift
import ExpoModulesCore
import WebKit

public class SimpleModule: Module {
  public func definition() -> ModuleDefinition {
    Constant("test") {
      return "Swift constant 1283"
    }
    
    Constant("testOtherFile") {
      let a = TestClass()
      return a.a;
    }

    Function("addNumbers") { (a: Double, b: Double) -> Int in
      return a + b
    }
    Class(TestClass.self) {
      Constructor { (a: Int) in
        TestClass(a)
      }

      Property("a") { (c: TestClass) in
        c.a
      }
    }
  }
}

public class TestClass{
  var a
}